exist-versioning-release.md

```diff
@@ -93,40 +93,121 @@ That version number is also copied into the `META-INF/MANIFEST.MF` file of any J
 
 ## Release Process
 
-This section details concrete steps for creating and publishing product releases. Each section here assumes you are starting with a clean Git checkout of the `develop` branch from [https://github.com/eXist-db/exist.git](https://github.com/eXist-db/exist.git).
+This section details concrete steps for creating and publishing product releases. 
 
-### Initiating Semantic Versioning
+### Release preprations and prerequisites 
 
-Version 3.0.0 was released before Semantic Versioning. The following steps will initiate Semantic Versioning for the remainder of the development phase of the next release, a new minor version to be called version 3.1.0:
+#### Required Software 
+Once development on a new stable version is complete, the following steps will prepare the version for release. For purposes of illustration, we will assume we are preparing the stable release of version 5.3.0.
+You will require a system with:
+
+* macOS
+* JDK 8
+* Maven 3.6.0+
+* Docker
+* GnuPG
+
+#### Required Keys, Credentials and Permissions
+
+##### GPG Key / Keystore and Credentials
+A private GPG key for signing release artifacts is required. To gather the required information run `gpg --list-secret-keys --keyid-format SHORT`
+
+```
+gpg --list-secret-keys --keyid-format SHORT
+/Users/username/.gnupg/pubring.kbx
+----------------------------------
+sec   rsa4096/12345678 2013-04-26 [SC]
+```
+
+This gives you the keystore you are using (here: `/Users/username/.gnupg/pubring.kbx`) and your key-id (here: `12345678` the part after rsa4096/). 
+
+**Important Note** 
+GnuPG is using an [optimized keystore since version 2.1](https://www.gnupg.org/documentation/manuals/gnupg/GPG-Configuration.html) which contains public and secret keys. Former GnuPG version had seperated keystore for public (~/.gnupg/pubring.gpg) and private keys (~/.gnupg/secring.gpg). 
```

Comment:
This should be moved before the commands (see comment above).

Reply:
Agreed, once its above the sample I think the discussion which keyring we show in the sample is not so important anymore, or is it? Point is since GnuPG 2.1 is .kbx thats why I would vote for the .kbx sample but I don't have strong feeling about it.. @adamretter what is printed if you run the command, only the pubring.gpg or secring.gpg and pubring.gpg ?